recipes/pdielec/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "PDielec" %}
+{% set version = "6.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 57c12cb29768dd9b49c7c17e12bf6570fa56a135a824f394a7538f011255f577
+
+build:
+  number: 0
+  entry_points:
+    - preader=PDielec.preader:main
+    - pdgui=PDielec.pdgui:main
+  script: "python -m pip install . -vv"
```

Comment:
using `{{ PYTHON }}` is usually more robust

Reply:
I tried this before, but got an error.  I will try again with noarch:python